apps/dot-cdn/src/app/app.component.html

```diff
@@ -2,58 +2,91 @@
+                        [ngStyle]="{ height: chartHeight }"
+                    ></dot-spinner>
+                    <ng-template #chartBlock>
+                        <h3 class="dot-cdn__tab-content-label">Bandwidth Served</h3>
+                        <p-chart
+                            #chart
+                            type="line"
+                            [data]="chartData"
+                            width="100%"
+                            [height]="chartHeight"
+                            [options]="options"
+                        ></p-chart>
+                    </ng-template>
                 </div>
             </div>
         </p-tabPanel>
         <p-tabPanel header="Flush Cache">
             <div class="dot-cdn__tab-content--contained">
-                <div class="dot-cdn__tab-content__row">
-                    <h3 class="dot-cdn__tab-content-label">Purge URL List</h3>
-                    <p>
-                        Purging an url list will remove the file from the CDN cache and re-download
-                        it from your server. Please enter the exact CDN URL of each individual file.
-                        Wildcard values are not supported.
-                    </p>
-                    <textarea [rows]="5" pInputTextarea></textarea>
-                    <button pButton label="Purge URL" disabled="true"></button>
-                </div>
-                <div class="dot-cdn__tab-content__row">
-                    <h3 class="dot-cdn__tab-content-label">Purge Pull Zone</h3>
-                    <p>
-                        Purging a pull zone will remove completely remove all of its files from
-                        cache and foce everything to be re-downloaded from your origin. If your zone
-                        uses a lot of storage, this ight send a large amount of traffic back to your
-                        server, so use with caution.
-                    </p>
-                    <button
-                        pButton
-                        label="Purge Pull Zone"
-                        class="p-button-danger p-button-outlined"
-                    ></button>
-                </div>
+                <form [formGroup]="purgeZoneForm">
+                    <div class="dot-cdn__tab-content__row">
+                        <h3 class="dot-cdn__tab-content-label">Purge URL List</h3>
+                        <p>
+                            Purging an url list will remove the file from the CDN cache and
+                            re-download it from your server. Please enter the exact CDN URL of each
+                            individual file. Wildcard values are not supported.
+                        </p>
+                        <textarea
+                            [rows]="5"
+                            pInputTextarea
+                            formControlName="purgeUrlsTextArea"
+                            (ngModelChange)="setUrlString($event)"
```

Comment:
```suggestion
```
You don't need to do this because you are using reactive forms, you will have the value of the form in `this.form.value`.

Reply:
@fmontes  I need to bind it to 'something' so I can listen the onChange and change the Button's UI accordingly

Reply:
NVM. As suggested, got it from `purgeZoneForm.get('purgeUrlsTextArea').value`